_rules/iframe-not-focusable-has-no-interactive-content-akn7bn.md

```diff
@@ -59,54 +59,54 @@ By setting the `tabindex` [attribute value][] of an `iframe` element to `-1` or
 
 #### Passed Example 1
 
-This `iframe` element contains no content that is part of [sequential focus navigation][].
+This `iframe` element contains a link that is not part of [sequential focus navigation][] because of its `tabindex`.
 
 ```html
-<iframe tabindex="-1" srcdoc="<h1>Hello world</h1>"></iframe>
+<iframe tabindex="-1" srcdoc="<a href='/' tabindex='-1'>Home</a>"></iframe>
```

Comment:
I agree that with the change, this is now Inapplicable and should be moved accordingly.